CHANGELOG.md

```diff
@@ -127,10 +127,41 @@ the APIs in these libraries are stable, and are ready for production use.
 - [Alloy DB](/google/cloud/alloydb/README.md)
 - [API Keys](/google/cloud/apikeys/README.md)
 
+### [BigQuery](/google/cloud/bigquery/README.md)
+
+- fix: retries for GetIamPolicy, TestIamPermissions ([#10957](https://github.com/googleapis/google-cloud-cpp/pull/10957))
```

Comment:
I would delete the library specific entries and leave the "Common Libraries" entry for this PR.... but your call.

Reply:
SGTM.  Will do.